frontend/subscriptions.js

```diff
@@ -0,0 +1,120 @@
+import { css } from 'glamor';
+import { appDidStart$, logger, routeDidEnter$ } from '@shopgate/engage/core';
+import { userDataReceived$, userDidLogout$ } from '@shopgate/engage/user';
+import { makeGetRoutePattern } from '@shopgate/pwa-common/selectors/router';
+import { sdkUrl, pagesWithoutWidget } from './config';
+import { getUserData } from './selectors';
+
+export default (subscribe) => {
+  const { style } = document.documentElement;
+  let ready = false;
+
+  // "live chat" -> userlike-tab. "UM" -> #uslk-button
+  css.global('#userlike-tab, #uslk-button', {
+    bottom: 'calc(10px + var(--safe-area-inset-bottom) + var(--footer-height) ) !important',
```

Comment:
As you see on video, it changes position on PDP when Sheet comes up, because of footer height

Reply:
done